skunkworks_crow/src/main/java/org/odk/share/views/ui/bluetooth/BluetoothListAdapter.java

```diff
@@ -69,7 +70,7 @@ public void onBindViewHolder(@NonNull final ViewHolder viewHolder, int position)
         int deviceBondState = device.getBondState();
         viewHolder.deviceName.setText(deviceName == null ? viewHolder.itemView.getResources().getString(R.string.bluetooth_instance_name_default) : deviceName);
         viewHolder.deviceAddress.setText(String.format("%s (%s)", deviceAddress,
-                deviceBondState == 10 ? context.getString(R.string.bluetooth_unpaired) : context.getString(R.string.bluetooth_paired)));
```

Comment:
Just a question what was this hardcoded 10 value?

Reply:
The same as `BOND_NONE`.